### PR TITLE
Add --organization option to `mix hex.package`

### PR DESCRIPTION
**Pull request by wojtekmach**

### CODE REVIEW

File: lib/mix/tasks/hex.package.ex [hunk 1]
```diff
@@ -128,7 +127,10 @@ defmodule Mix.Tasks.Hex.Package do
         tarball
 
       {:error, reason} ->
-        Mix.raise(reason)
+        Mix.raise(
+          "Downloading " <>
+            Hex.Repo.tarball_url(repo, package, version) <> " failed:\n\n" <> reason
```
Comment (wojtekmach) [inline lib/mix/tasks/hex.package.ex]: this way the error message is imho a little bit more useful, here's how it looks like:

image

File: lib/mix/tasks/hex.docs.ex [hunk 2]
```diff
@@ -32,7 +32,7 @@ defmodule Mix.Tasks.Hex.Docs do
   ## Command line options
 
     * `--module Some.Module` - Open a specified module documentation page inside desired package
-    * `--organization ORGANIZATION` - The organization the package belongs to
+    * `--organization ORGANIZATION` - Set this for private packages belonging to an organization
```
Comment (ericmj) [inline lib/mix/tasks/hex.docs.ex]: Awesome!

File: lib/mix/tasks/hex.package.ex [hunk 3]
```diff
@@ -66,11 +64,12 @@ defmodule Mix.Tasks.Hex.Package do
   ## Command line options
 
   * `--unpack` - Unpacks the tarball after fetching it
+  * `--organization ORGANIZATION` - The organization the package belongs to
```
Comment (ericmj) [inline lib/mix/tasks/hex.package.ex]: Also need it here though :)

Comment (wojtekmach) [reply lib/mix/tasks/hex.package.ex]: oops :D

**Pull request merged**